ingestion-beam/src/test/java/com/mozilla/telemetry/decoder/DeduplicateTest.java

```diff
@@ -39,8 +39,8 @@ public void testOutput() {
 
     // Create new PubsubMessage with element as document_id attribute
     final MapElements<String, PubsubMessage> mapStringsToId = MapElements
-        .into(new TypeDescriptor<PubsubMessage>() {
-        }).via(element -> new PubsubMessage(new byte[0], ImmutableMap.of("document_id", element)));
+        .into(TypeDescriptor.of(PubsubMessage.class))
+        .via(element -> new PubsubMessage(new byte[0], ImmutableMap.of("document_id", element)));
```

Comment:
We remove a few instances of `new TypeDescriptor` with the simpler `TypeDescriptor.of`